modules/nw-networkpolicy-about.adoc

```diff
@@ -8,8 +8,11 @@
 = About network policy
 
 In a cluster using a Kubernetes Container Network Interface (CNI) plug-in that supports Kubernetes network policy, network isolation is controlled entirely by `NetworkPolicy` objects.
+
 In {product-title} {product-version}, OpenShift SDN supports using network policy in its default network isolation mode.
 
+When using the OpenShift SDN cluster network provider, the egress network policy as specified by the `egress` field is now supported.
```

Comment:
To avoid passive voice, consider revising it to "The OpenShift SDN cluster network provider now supports the egress network policy as specified by the 'egress' field" if it doesn't change the meaning of the sentence.  
Rest LGTM.